srcpkgs/brother-mfc5490cn-lpr/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'brother-mfc5490cn-lpr'
+pkgname=brother-mfc5490cn-lpr
+version=1.1.2
+revision=2
+short_desc="LPR driver for Brother MFC-5490CN printer/scanner"
+maintainer="Henner M. Kruse <github@hmkruse.de>"
+license="proprietary"
+homepage="http://support.brother.com/g/b/index.aspx"
+distfiles="http://download.brother.com/welcome/dlf006156/mfc5490cnlpr-${version}-2.i386.deb"
+checksum=4a361dea157994cd825dbeba529ff560b279a591643ce8a0e0b1acce72f0bac5
+repository="nonfree"
+only_for_archs="i686"
+lib32mode="full"
+depends="a2ps ghostscript"
+nopie=yes
+#mutable_files="/opt/brother/Printers/mfc5490cn/inf/brmfc5490cnrc"
```

Comment:
Remove this line.

Reply:
And please squash the xlint fixes into their respective commits. This PR should have 2 commits.